### ISSUE TITLE
yii\db\command execute() don't handle errors on multiple queries

### ISSUE THREAD

**Issue by tomaszkane** [yiisoft/db#79]
### What steps will reproduce the problem?
Try to import db schema (from mysqldump or PhpMyAdmin schema export) with some error on non-first query. In example, change "int" column type in some table to "intFOO" to cause mysql error. 

Schema:
```mysql
SET FOREIGN_KEY_CHECKS=0;
DROP TABLE IF EXISTS `auth_assignment`;
CREATE TABLE `auth_assignment` (
  `item_name` varchar(64) COLLATE utf8_unicode_ci NOT NULL,
  `user_id` intFOO(10) UNSIGNED NOT NULL,
  `created_at` int(11) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
```

Example code (console action):
```php
public function actionReset()
{
    $schemaFile = 'some/path/schema.sql';
    try {
        $command = Yii::$app->db->createCommand(file_get_contents($schemaFile));
        $command->execute();

        Console::output('Schema imported.');
        return self::EXIT_CODE_NORMAL;
    } catch (\Exception $e) {
        Console::error($e->getMessage());
    }
    return self::EXIT_CODE_ERROR;
}
```

### What is the expected result?

Execution ends with proper exception.

### What do you get instead?

"Schema imported." on stdout - no exception thrown, or error reported.

### Additional info

Read this question and approved answer: [How to check if db execute sql fails?](https://stackoverflow.com/questions/46178560/how-to-check-if-db-execute-sql-fails)

I believe that Yii should handle this case?

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.12
| PHP version      | 7.1.x
| Operating system | Kubuntu 17.04

**Comment by FantomX1** [comment]
Delimiters in MySQL
https://stackoverflow.com/a/10259528/3419535

> 
> Attempting to use DELIMITER with a client that doesn't support it will cause it to be sent to the server, which will report a syntax error. For example, using PHP and MySQLi:
> 
> $mysqli = new mysqli('localhost', 'user', 'pass', 'test');
> $result = $mysqli->query('DELIMITER $$');
> echo $mysqli->error;
> Errors with:
> 
> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'DELIMITER $$' at line 1
>

**Comment by thomaswruss** [comment]
I had the same problem if I execute a procedure that calls more than one procedure. If there is an error, the error will not be raised in Yii. But if I execute just that one procedure, the error will be raised. So this seems to be a bug of Yii. Any suggestion how to detect the errors in a multiple procedure call?

**Comment by samdark** [comment]
> Any suggestion how to detect the errors in a multiple procedure call?

Yes. See https://github.com/yiisoft/db/issues/79#issuecomment-330821059

**Comment by tomaszkane** [comment]
I also link a solution in Additional info in this issue: https://stackoverflow.com/a/46280130/1829368
@thomaswruss read topic before ask.
